app/javascript/utilities/keyMap.js

```diff
@@ -0,0 +1,35 @@
+export const KeyMap = {
+  1: 0,
+  2: 1,
+  3: 2,
+  4: 3,
+  q: 4,
+  w: 5,
+  e: 6,
+  r: 7,
+  a: 8,
+  s: 9,
+  d: 10,
+  f: 11,
+  z: 12,
+  x: 13,
+  c: 14,
+  v: 15,
+  // Maj
+  '!': 0,
+  '@': 1,
+  '#': 2,
+  $: 3,
+  Q: 4,
```

Comment:
What do you think about using `e.key.toLowerCase()`, so that we don't have to handle the uppercase? 🤔

Reply:
I think it won't work with the first row 😇 
<img width="159" alt="image" src="https://user-images.githubusercontent.com/77609814/191405219-daa8f175-100e-4883-a970-41c5e42947fa.png">